Mutagen.Bethesda.Analyzers.Skyrim/CircularLeveledIListAnalyzer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Results;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Plugins;
+using Mutagen.Bethesda.Plugins.Cache;
+using Mutagen.Bethesda.Plugins.Records;
+namespace Mutagen.Bethesda.Analyzers.Skyrim;
+
+public partial class CircularLeveledListAnalyzer
+{
+    public IEnumerable<TopicDefinition> Topics => [CircularLeveledItem, CircularLeveledNpc, CircularLeveledSpell];
+
+    private static RecordAnalyzerResult FindCircularList<T>(T leveled, Func<T, IEnumerable<FormKey>> nestedEntriesSelector, ILinkCache linkCache)
+        where T : class, IMajorRecordGetter
+    {
+        var report = new RecordAnalyzerResult();
```

Comment:
Yeah seems like I forgot to add that properly. This was one of the few analyzers I didn't test properly because I didn't have any plugin with this issue at hand.